_dashboards/visualize/viz-index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Visualize
-nav_order: 4
+title: Building data visualizations with Visualize
```

Comment:
Can we drop 'with Visualize'?

Reply:
Visualize is the application on the UI.

Reply:
![Screen Shot 2023-02-03 at 10 03 26 AM](https://user-images.githubusercontent.com/105296784/216716282-2c47a9d9-c62a-49ed-9891-0c4831455613.png)

Reply:
@hdhalter Changes made as suggested.

Reply:
TOC updates
![Screen Shot 2023-02-03 at 2 58 35 PM](https://user-images.githubusercontent.com/105296784/216718942-ff0cacb8-18d9-4a8a-b709-65b803ecc800.png)